cfme/tests/configure/test_access_control.py

```diff
@@ -1651,7 +1650,21 @@ def test_tenantadmin_user_crud():
             must be created by superadministrator. In 5.5.0.13 after giving additional permissions
             to tenant_admin,able to create new roles
     """
-    pass
+    with new_tenant_admin:
+        navigate_to(appliance.server, 'LoggedIn')
+        assert appliance.server.current_full_name() == new_tenant_admin.name
+
+        group = appliance.collections.groups.instantiate(
```

Comment:
Isn't this duplicating the group entity that's already available at `new_tenant_admin.groups[0]`, just without setting role or tenant?

```
(Pdb) new_tenant_admin.groups[0]
Group(description='tenant_grp_H3b7QMIGL3', role='EvmRole-tenant_administrator_copy', tenant='My Company/child_tenantKeUAt0WIBK', ldap_credentials=None, user_to_lookup=None, tag=None, host_cluster=None, vm_template=None)
```